Task4.py

```diff
@@ -0,0 +1,19 @@
+"""
```

Comment:
Still present linter errors:
(venv) [antoha@dlhn test_project]$ ./bin/lint /home/antoha/PycharmProjects/test_project/T-HOMETASKS/L04/l04_h01_InnaKopeichikova.py
/home/antoha/PycharmProjects/test_project/T-HOMETASKS/L04/l04_h01_InnaKopeichikova.py:1:1: D205 1 blank line required between summary line and description
/home/antoha/PycharmProjects/test_project/T-HOMETASKS/L04/l04_h01_InnaKopeichikova.py:1:1: D208 Docstring is over-indented
/home/antoha/PycharmProjects/test_project/T-HOMETASKS/L04/l04_h01_InnaKopeichikova.py:5:80: E501 line too long (101 > 79 characters)
/home/antoha/PycharmProjects/test_project/T-HOMETASKS/L04/l04_h01_InnaKopeichikova.py:18:37: W292 no newline at end of file

Reply:
For some reason my linter shows me only these errors:

(venv) inna@Innas-MacBook  Project 1 % ./bin/lint Python-Hillel/Task4.py
Python-Hillel/Task4.py:1:1: D208 Docstring is over-indented
Python-Hillel/Task4.py:6:80: E501 line too long (101 > 79 characters)

If I run only the flake8 command :
(venv) inna@Innas-MacBook  Project 1 % flake8 Python-Hillel/Task4.py
Python-Hillel/Task4.py:0:1: WPS102 Found incorrect module name pattern
Python-Hillel/Task4.py:1:1: D208 Docstring is over-indented
Python-Hillel/Task4.py:6:80: E501 line too long (101 > 79 characters)
Python-Hillel/Task4.py:10:1: WPS111 Found too short name: w < 2
Python-Hillel/Task4.py:10:4: WPS111 Found too short name: x < 2
Python-Hillel/Task4.py:10:7: WPS111 Found too short name: y < 2
Python-Hillel/Task4.py:10:10: WPS111 Found too short name: z < 2
Python-Hillel/Task4.py:13:5: WPS421 Found wrong function call: print
Python-Hillel/Task4.py:15:5: WPS421 Found wrong function call: print
Python-Hillel/Task4.py:17:5: WPS421 Found wrong function call: print
Python-Hillel/Task4.py:19:5: WPS421 Found wrong function call: print

Why could there be a difference between mine and your outcome of linter?
(My lint file content is : flake8 --ignore=WPS,DAR $1,
and setup file: [flake8]
min-name-length=3)